src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Message.java

```diff
@@ -32,6 +33,12 @@
         .takeWhile(segment -> !segment.startsWith(MessageTrailer.KEY))
         .collect(toList()));
 
+    @Getter(lazy = true)
+    private final Optional<PerformerNameAndAddress> performerNameAndAddress =
+        extractOptionalSegment(PerformerNameAndAddress.KEY_QUALIFIER)
+            .map(PerformerNameAndAddress::fromString);
+
+
```

Comment:
Careful of extra blank lines; some style tools complain about it
```suggestion

```

Reply:
Done in b4a72f8